intel_extension_for_transformers/llm/runtime/graph/__init__.py

```diff
@@ -141,6 +143,11 @@ def quant_model(self, model_type, model_path, out_path, **quant_kwargs):
         self.__import_package(model_type)
         self.module.Model.quant_model(model_path=model_path, out_path=out_path, **quant_kwargs)
 
+    def inference(self, audio_file):
```

Comment:
```suggestion
def __call__(self, input_ids, reinit=False, audio_file=None, **kwargs):
        if self.model is None:
            self.init_from_bin(self.model_type, self.bin_file, **kwargs)
            self.generate_round = 0
        elif reinit:
            self.model.reinit()
            self.generate_round = 0
        if input_ids is not None:
            return self.model.evaluate(input_ids.tolist())
         if audio_file is not None:
             return self.model.inference(audio_file)
```
we can also add `model_type` liked args in kwargs for different models inference with different types

Reply:
[700bbc2](https://github.com/intel/intel-extension-for-transformers/pull/809/commits/700bbc2c4b71bfabbed14331da0eec6edcbbe841) looks good to me